core/drivers/gic.c

```diff
@@ -858,6 +858,30 @@ void gic_dump_state(void)
 	}
 }
 
+void gic_spi_release_to_ns(size_t it)
+{
+	struct gic_data *gd = &gic_data;
+	size_t idx = it / NUM_INTS_PER_REG;
+	uint32_t mask = 1 << (it % NUM_INTS_PER_REG);
```

Comment:
Please use the `BIT32()` macro instead. I know we're doing a bit of both in this file, but using `BIT32()` is the preferred way.

Reply:
done